_partials/windows_ubuntu.md

```diff
@@ -2,6 +2,16 @@
 
 ### Installation
 
+:information_source: The following instructions depend on your version of Windows. Please execute only the instructions corresponding to your version :point_down:
+
+#### Windows 11
+
+If you are running Windows 11, after restarting you computer, you should see a terminal window saying WSL is resuming the Ubuntu installation process. When it's done, Ubuntu will be launched.
```

Comment:
Don't we need to `wsl --install -d Ubuntu`?

Reply:
From what I've read [here](https://pureinfotech.com/install-wsl-windows-11/), `wsl --install` actually installs Ubuntu by default. Ubuntu seems to be the default option of the `-d` flag and thus it's not necessary to pass it: 
> In addition, the command also installs the latest Ubuntu distro by default.